JNWCollectionView/JNWCollectionViewFramework.m

```diff
@@ -131,9 +137,23 @@ - (id)initWithCoder:(NSCoder *)aDecoder {
 	return self;
 }
 
+#pragma mark Notifications
+
+- (void)preferredScrollerStyleDidChangeNotification:(NSNotification *)note {
+	NSView *documentView = self.documentView;
+	
+	if (!CGSizeEqualToSize(NSSizeToCGSize(documentView.frame.size), NSSizeToCGSize(self.contentSize))) {
```

Comment:
As this library is 64-bit only, there's no need to bridge between `NSRect` and `CGRect`. They can be used interchangeably. So it could just look like this:

```objc
if (!CGSizeEqualToSize(documentView.frame.size, self.contentSize)) {
```